src/main/java/org/owasp/html/HtmlTagSkipType.java

```diff
@@ -0,0 +1,34 @@
+package org.owasp.html;
+
+public enum HtmlTagSkipType {
+  SKIP_BY_DEFAULT(true),
+  SKIP(true),
+  DO_NOT_SKIP(false),
+  NONE(false);
+
+  private final boolean skipAvailability;
+
+  HtmlTagSkipType(boolean skipAvailability) {
+    this.skipAvailability = skipAvailability;
+  }
+
+  public HtmlTagSkipType and(HtmlTagSkipType s) {
+    if (this == s || s == SKIP_BY_DEFAULT) {
+      return this;
+    }
+
+    if (s == DO_NOT_SKIP) {
+      return s;
+    }
+
+    if (s == NONE) {
+      return this;
+    }
+
+    return SKIP;
```

Comment:
You said `SKIP_BY_DEFAULT.and(DO_NOT_SKIP) == SKIP`
but I think `SKIP_BY_DEFAULT.and(DO_NOT_SKIP) == DO_NOT_SKIP`  is right. 
because of testHtmlTagSkipPolicy2 code.
If beforePolicy set nothing, and afterPolicy set `.allowWithoutAttributes("span")`,
I think the final result is `DO_NOT_SKIP`.
Am I wrong?
```java
  // Default Skip Tag
  // beforePolicy : X
  // afterPolicy : allow
  @Test
  public void testHtmlTagSkipPolicy2() {
    PolicyFactory beforePolicy = new HtmlPolicyBuilder()
            .allowElements("span")
            .toFactory();

    String spanTagString = "<span>Hi</span>";
    String resultString = beforePolicy.sanitize(spanTagString);
    assertEquals("Hi", resultString);

    PolicyFactory afterPolicy = beforePolicy.and(new HtmlPolicyBuilder()
            .allowElements("span")
            .allowWithoutAttributes("span")
            .toFactory());

    resultString = afterPolicy.sanitize(spanTagString);
    assertEquals("<span>Hi</span>", resultString);
  }
```

<img width="878" alt="스크린샷 2020-05-30 오후 11 03 03" src="https://user-images.githubusercontent.com/7060510/83330578-da6ea800-a2ca-11ea-9939-5e16f12f8b9b.png">

<img width="595" alt="스크린샷 2020-05-30 오후 11 07 34" src="https://user-images.githubusercontent.com/7060510/83330579-de9ac580-a2ca-11ea-9955-63dccd1e65c9.png">